src/lib/jsonParse/jsonRequest.cpp

```diff
@@ -258,7 +258,7 @@ std::string jsonTreat
   if (res != "OK")
   {
     std::string details = reqP->keyword + ": " + res;
-    alarmMgr.badInput(clientIp, details);
+    alarmMgr.badInput(clientIp, details, "");
```

Comment:
std::string details = reqP->keyword;
      alarmMgr.badInput(clientIp, details, res);

Reply:
Ok

Reply:
Ok

Reply:
Fixed in 10cdfce2c